#goal definition
int32 target_node_id
---
#result definition
bool success
---
#feedback
int32[] route_node_ids
int32[] route_edge_ids
